src/omega_edit/interface.html

```diff
@@ -181,6 +181,13 @@
               />
               <span style="margin-left: 0px" class="checkmark"></span>
             </label>
+            <label
+              style="margin-left: 142%; margin-top: -16px; width: 105px"
+              class="container"
+              >Case Sensitive?
+              <input type="checkbox" id="caseSensitive" />
+              <span style="margin-left: 0px" class="checkmark"></span>
+            </label>
```

Comment:
The label should be Case Insensitive, the id should be caseInsensitive, and the default should be unchecked.

Ideally the UI would get a byte frequency profile from the server, then only allow this setting if the profile indicates that the data is all ASCII.